snapcraft/internal/pluginhandler.py

```diff
@@ -650,21 +650,34 @@ def _migratable_filesets(fileset, srcdir):
     return snap_files, snap_dirs
 
 
+def _create_dirs(srcdir, dstdir, follow_symlinks=False):
+    dir_stat = os.stat(srcdir, follow_symlinks=follow_symlinks)
+    uid = dir_stat.st_uid
+    gid = dir_stat.st_gid
+    os.makedirs(dstdir, exist_ok=True)
+    try:
+        os.chown(dstdir, uid, gid, follow_symlinks=follow_symlinks)
+    except PermissionError as e:
+        logger.warning(e)
```

Comment:
Will this actually be informative? Or does it only say "permission denied"? If it's the latter, it might be worth adding the directory name to the warning (e.g. "unable to chown {dstdir}: {error}").